app/api/v2/handlers/operation_api.py

```diff
@@ -34,39 +34,53 @@ def add_routes(self, app: web.Application):
         router.add_get('/operations/{id}/potential-links', self.get_potential_links)
         router.add_get('/operations/{id}/potential-links/{paw}', self.get_potential_links_by_paw)
 
-    @aiohttp_apispec.docs(tags=['operations'])
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Retrieve operations',
+                          description='Retrieve all CALDERA operations from memory.  Use fields from the '
+                                      '`BaseGetAllQuerySchema` in the request body to filter.')
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
     @aiohttp_apispec.response_schema(OperationSchema(many=True, partial=True))
     async def get_operations(self, request: web.Request):
         operations = await self.get_all_objects(request)
         return web.json_response(operations)
 
-    @aiohttp_apispec.docs(tags=['operations'])
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Retrieve an operation by operation id',
+                          description='Retrieve one CALDERA operation from memory based on the operation id (String '
+                                      'UUID).  Use fields from the `BaseGetOneQuerySchema` in the request body to add '
+                                      '`include` and `exclude` filters.')
```

Comment:
For the endpoints that contain a parameter in the url (GET operation by ID, PATCH, PUT, and DELETE) you can add the following to the `aiohttp_apispec.docs` decorator to describe the url parameter:
```
parameters=[{
  'in': 'path',
  'name': 'id',
  'schema': {'type': 'string'},
  'required': 'true',
  'description': 'UUID of the Operation object to be retrieved.'
}],
```